src/components/v5/common/ActionSidebar/partials/ActionSidebarContent/ActionSidebarContent.tsx

```diff
@@ -73,12 +140,17 @@ const ActionSidebarFormContent: FC<ActionSidebarFormContentProps> = ({
           {!selectedAction && (
             <PopularActions
               setSelectedAction={(action) =>
-                form.setValue(ACTION_TYPE_FIELD_NAME, action)
+                setValue(ACTION_TYPE_FIELD_NAME, action)
               }
             />
           )}
           <ActionButtons
-            isActionDisabled={!userHasPermissions || !selectedAction}
+            isActionDisabled={
+              !userHasPermissions ||
+              !selectedAction ||
+              noReputationError ||
+              noPermissionsError
+            }
           />
```

Comment:
I belive the UX of these forms should be to never disable the CTA button but to show the errors when the user tries to submit the incorrect values. Can you confirm @melyndav 

![image](https://github.com/JoinColony/colonyCDapp/assets/32598350/8fdc1811-c8f3-4222-bb0f-2da42cff4268)

Reply:
> ld be to never disable the CTA button but to show the errors when the user tries to submit the incorrect values. Can

This is the correct behaviour, as you said @thomassdk. The CTA is never disabled when a user selects an action type.

The only time it's disabled is when the user opens the sidepanel from the navigation and there are no actions selected.